### DISCUSSION TITLE
How to select store and product?

### DISCUSSION THREAD

**Discussion by DerDegu** [jef/streetmerchant#1643] [Help]
Ok, I found it ^^

**Comment by MrKomodoDragon** [comment]
Could you please tell me how you did this? I was wondering too.

**Reply by DerDegu** [reply]
store:

Put the names of the stores in the ".env" at "STORES="(line 102), for example "STORES=spielegrotte,saturn,otto".
You find the names at "streetmerchant\src\store\model".


product:

Put the name of the pruduct or brand (for the ps5 I use the brand) in the ".env" at "SHOW_ONLY_MODELS="(line 96) or "SHOW_ONLY_BRANDS="(line 95), for example "SHOW_ONLY_BRANDS=sony".

You find the correct names for the products and brands in the "xxx.ts" files at "streetmerchant\src\store\model".

Sorry for my bad english ^^

**Reply by MrKomodoDragon** [reply]
Thank you!
